### PR TITLE
Small grammatical changes to Sampling document

### PR DESCRIPTION
**Pull request by jackgopack4**
Some small suggested changes to improve grammar/clarity of Sampling doc without changing meaning or content

### CODE REVIEW

File: content/en/docs/concepts/sampling/index.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-  Learn about sampling, and the different sampling options available in
+  Learn about sampling and the different sampling options available in
   OpenTelemetry.
 weight: 80
 ---
 
-With distributed tracing, you observe requests as they move from one service to
-another in a distributed system. It’s superbly practical for a number of
+With distributed tracing, you can observe requests as they move from one service
+to another in a distributed system. It is superbly practical for a number of
```
Comment (tiffany76) [inline content/en/docs/concepts/sampling/index.md:10]: We are generally okay with contractions, so you could leave this as "It's".

File: content/en/docs/concepts/sampling/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ want to sample. You might want to customize your sampling strategy to:
 
 ## Terminology
 
-It's important to use consistent terminology when discussing sampling. A trace
+It is important to use consistent terminology when discussing sampling. A trace
```
Comment (tiffany76) [inline content/en/docs/concepts/sampling/index.md:35]: Ditto here.

File: content/en/docs/concepts/sampling/index.md [hunk 3]
```diff
@@ -53,8 +53,8 @@ inspecting the trace as a whole.
 
 For example, the most common form of head sampling is
 [Consistent Probability Sampling](/docs/specs/otel/trace/tracestate-probability-sampling/#consistent-probability-sampling).
-It may also be referred to as Deterministic Sampling. In this case, a sampling
-decision is made based on the trace ID and a desired percentage of traces to
+This may also be referred to as Deterministic Sampling. In this case, a sampling
```
Comment (tiffany76) [inline content/en/docs/concepts/sampling/index.md:56]: If you're up for it, we are trying to root out the use of "may" in the docs, so you could change any occurrences on this page to "might" and earn our sincere gratitude!

File: content/en/docs/concepts/sampling/index.md [hunk 4]
```diff
@@ -105,9 +105,9 @@ There are three primary downsides to tail sampling today:
   tail sampling must be stateful systems that can accept and store a large
   amount of data. Depending on traffic patterns, this can require dozens or even
   hundreds of nodes that all utilize resources differently. Furthermore, a tail
-  sampler may need to "fall back" to less computationally-intensive sampling
+  sampler may need to "fall back" to less computationally intensive sampling
```
Comment (tiffany76) [inline content/en/docs/concepts/sampling/index.md:108]: Nice catch!

File: content/en/docs/concepts/sampling/index.md [hunk 5]
```diff
@@ -117,7 +117,7 @@ There are three primary downsides to tail sampling today:
 
 Finally, for some systems, tail sampling may be used in conjunction with Head
 Sampling. For example, a set of services that produce an extremely high volume
-of trace data may first use head sampling to only sample a small percentage of
+of trace data may first use head sampling to sample only a small percentage of
```
Comment (tiffany76) [inline content/en/docs/concepts/sampling/index.md:120]: :heart:

**Review by tiffany76 — APPROVED** [review]
Thanks for proofreading this page. You made some excellent changes.

I made a couple suggestions, but this PR looks good to me!

File: content/en/docs/concepts/sampling/index.md (hunk 3 above)
Comment (jackgopack4) [reply content/en/docs/concepts/sampling/index.md:56]: it is referred to as Deterministic Sampling, so I changed this one to "is" but will update the others to use "might"

File: content/en/docs/concepts/sampling/index.md (hunk 1 above)
Comment (jackgopack4) [reply content/en/docs/concepts/sampling/index.md]: reverted, thanks

File: content/en/docs/concepts/sampling/index.md (hunk 2 above)
Comment (jackgopack4) [reply content/en/docs/concepts/sampling/index.md]: reverted, thanks

**Comment by chalin** [comment]
Thanks all!

**Pull request merged**